hydra/grammar/OverrideParser.g4

```diff
@@ -47,12 +47,17 @@ function: ID POPEN (argName? element (COMMA argName? element )* )? PCLOSE;
 
 // Data structures.
 
-listValue: BRACKET_OPEN                          // [], [1,2,3], [a,b,[1,2]]
-    (element(COMMA element)*)?
-BRACKET_CLOSE;
-
+listValue: BRACKET_OPEN sequence? BRACKET_CLOSE;                          // [], [1,2,3], [a,b,[1,2]]
 dictValue: BRACE_OPEN (dictKeyValuePair (COMMA dictKeyValuePair)*)? BRACE_CLOSE;  // {}, {a:10,b:20}
-dictKeyValuePair: ID COLON element;
+dictKeyValuePair: (ID | interpolation) COLON element;
+sequence: element (COMMA element)*;
+
+// Interpolations.
+
+interpolation: interpolationNode | interpolationResolver;
+interpolationNode: INTER_OPEN DOT* configKey (DOT configKey)* INTER_CLOSE;
```

Comment:
add tests for relative interpolations, I think it's not covered here.

Reply:
True, my big list of tests from OmegaConf doesn't have examples of nested interpolations. However I ran into an issue when adding some -- will need more time to look into it (this could have to wait until next week).

Reply:
it could be in a followup. relative interpolations are very new in OmeaConf.